test/e2e/templates/basic-dev.tpl

```diff
@@ -1,15 +1,8 @@
 {{#> devLayout}}
-var options = {
-  'baseUrl': '{{{WIDGET_TEST_SERVER}}}'
-};
-var oktaSignIn = new OktaSignIn(options);
+var oktaSignIn;
 
-oktaSignIn.renderEl(
-  { el: '#okta-login-container' },
-  function (res) {
-    if (res.status === 'SUCCESS') {
-      res.session.setCookieAndRedirect(options.baseUrl + '/app/UserHome');
```

Comment:
Sanity check, do we still have a test that asserts that this flow is possible? (Do set cookie and redirect after the .renderEl() success handler called)

Reply:
We do. Those are still covered using the `basic` template